src/elements/CardElement.tsx

```diff
@@ -56,3 +59,9 @@ export const CardElement: FC<CardElementProps> = ({
 
   return <div id={id} ref={wrapperRef} />;
 };
+
+export const CardElement = React.forwardRef<ICardElement, CardElementProps>(
+  (props, ref) => <CardElementC {...props} elementRef={ref} />
```

Comment:
Does this function need a `displayName` for [debugging purposes](https://reactjs.org/docs/react-component.html#displayname)?

Same for other components below

Reply:
Any suggestions? `CardElementFR` ..etc?

Reply:
`CardElement` should suffice. The problem here is that the exported function is the result from `forwardRef`. I believe in debug the internal component that takes the ref will be anonymous, possibly leading to confusion.
We can test and see if that is the case using React Dev Tools